.github/workflows/all-embedded.yml

```diff
@@ -0,0 +1,33 @@
+# Tests for embedded platforms
+name: CI (embedded platforms)
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 8 * * 6'
```

Comment:
What is this? Are you setting up a cronjob to regularly run this CI job?

Reply:
I just copied this from the `all-misc.yml` and `all-targets.yml`, which are the two other top-level CI jobs. I can remove it if you'd like.

Reply:
We use CRON to do weekly (at 08:00 on Saturday), to run tests on macOS runners. We don't need it for the embedded platforms, so we should remove this.

Reply:
Aha, sorry, I just hadnt seen this before in the CI. Makes sense

Reply:
But, should be removed from embedded targets yes